### DISCUSSION TITLE
Sharing equipment on a local network through a link accessible from the Internet

### DISCUSSION THREAD

**Discussion by informaticavera** [Ylianst/MeshCentral#4815] [Q&A]
hello, we have meshcentral installed in our offices. This instance is not accessible from the internet and is used to provide support to local computers.

Meshcentral is a critical part of our infrastructure and that's why you don't publish it abroad in the dmz, our meshcentral is only accessible from our local network.

We need to have other software to allow the connection from the internet to our local computers, such as anydesk.

Is there any way to allow our computers to be only accessible from the Internet through the link that is generated in the "shared device" option?

Thanks

**Comment by dinger1986** [comment]
Using a share link from the same local server?

**Comment by informaticavera** [comment]
the local server is not accessible from the internet. Generates a link with a url of the type https://localserver/sharing?c=

And from the internet you can't access localserver

My question is aimed at whether it is possible to create another instance of meshcentral in a dmz that is only in charge of routing this type of traffic. A kind of gateway or virtual meshcentral router.

I don't know if meshcentral has any for this case or my only option is to put my local server facing the internet

**Comment by informaticavera** [comment]
It's not possible? Does anyone come up with something?
